eventually/static/src/containers/registerLogin/SetNewPassword.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {withRouter} from 'react-router-dom';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import {putNewPasswordService} from './registrationService';
+
+const style = {
+    margin: 12,
+};
+
+const errorStyle={
+    position: 'absolute',
+    bottom : '-10px'
+};
+
+class SetNewPassword extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            password: '',
+            confirmPassword: '',
+            token: this.props.match.params.token,
+            error:'',
+        };
+    }
+
+    handlePassword = event => {
+        this.setState({password: event.target.value});
+    };
+
+    handleConfirmPassword = event => {
+        this.setState({confirmPassword: event.target.value});
+    };
+
+    handleSubmit = event => {
+        if (this.state.password == this.state.confirmPassword){
+            const newPassword = this.state.password ;
+            const token = this.state.token;
+            putNewPasswordService(token, newPassword).then((response) => {
+                this.props.history.push('/login');
+            });
+        }
+        else {
+            this.setState({error: 'Passwords does not match'});}
```

Comment:
curly braces move in a new line

Reply:
added validators and disable/able button